src/express.ts

```diff
@@ -335,6 +341,10 @@ export class InngestCommHandler {
       sdk: this.sdkHeader[1],
       v: "0.1",
     };
+
+    // Calculate the checksum of the body... without the checksum itself being included.
+    body.hash = shajs("sha256").update(JSON.stringify(body)).digest("hex");
```

Comment:
It might be a good idea to do a checksum of `functions` instead of the entire `body`. We talked in the past about adding a checksum in the body for registering ([code](https://github.com/inngest/inngest-js/blob/81a89539f0c13ebd5667be75f14f62a216eaa3b3/src/express.ts#L344-L362)) so the Inngest API could determine if any functions have changed (Future "deployments" feature). What do you think is better? checksum of just the functions (or the other fields which likely won't change.

Reply:
To be honest, I think it's safer to re-deploy if any of the JSON changes (barring hash), such as the SDK versions, deploy type, etc.  Though some of these attributes are in the function config not all of them are.  This means that we can properly show deploy information to users as they upgrade (or don't upgrade) the SDK — which will be important with breaking changes in the future.